apps/files/src/components/FileDetails.vue

```diff
@@ -83,6 +83,16 @@ export default {
     },
     activeTabComponent () {
       return this.fileSideBarsEnabled.find(sidebar => sidebar.app === this.activeTab)
+    },
+    privateLinkOfHighlightedFile () {
+      if (!this.highlightedFile) {
+        return false
+      }
+      if (this.highlightedFile.isMounted()) {
+        const file = encodeURIComponent(this.highlightedFile.name)
+        return window.location.href.split('?')[0] + `?scrollTo=${file}`
```

Comment:
This will need to be edited to also enable navigating directly into folder in case it is one. Same for the "classic" private link with id. But that is a task for different PR.

Reply:
Was discussed with @hodyroff that the feedback for this was negative and that it should be aligned with how old webUI does it. Probably better if PM can clarify and decide the behaviour here @pmaier1